src/coreclr/pal/src/thread/context.cpp

```diff
@@ -436,7 +462,7 @@ void CONTEXTToNativeContext(CONST CONTEXT *lpContext, native_context_t *native)
 #undef ASSIGN_REG
 
 #if !HAVE_FPREGS_WITH_CW
-#if HAVE_GREGSET_T || HAVE_GREGSET_T
+#if (HAVE_GREGSET_T || HAVE_GREGSET_T) && !defined(HOST_S390X)
```

Comment:
I have just noticed this #if has a wrong condition (event before your change). Since you are touching this code, could you please fix it? It should be 
```c++
#if (HAVE_GREGSET_T || HAVE___GREGSET_T) && !defined(HOST_S390X)
```
Actually, the related #endif comment need to be updated to match your change (and this fix) to
```c++
#endif // HAVE_GREGSET_T || HAVE___GREGSET_T || !HOST_S390X
```

Reply:
Fixed.